src/components/RangeSlider/components/DualThumb/DualThumb.tsx

```diff
@@ -253,7 +326,8 @@ export default class DualThumb extends React.Component<Props, State> {
   }
 
   @autobind
-  private handleMouseDownThumbUpper() {
+  private handleMouseDownThumbUpper(event: MouseEvent) {
+    if (event.button !== 0) return;
```

Comment:
`if (event.button > 0)` ? 

I'm not clear on what this is doing.

Reply:
This is looking to see that the primary mouse button is the source of the click. The primary button should always return 0, so we want this specific check.

Reply:
Yep the JS API for the mouse button event just sucks